test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -253,3 +254,124 @@ func schedulePods(config *testConfig) int32 {
 		time.Sleep(1 * time.Second)
 	}
 }
+
+func (nodeAffinity *nodeAffinity) mutateNode(numNodes int) []testutils.CountToStrategy {
+	numGroups := nodeAffinity.numGroups
+	nodeAffinityKey := nodeAffinity.nodeAffinityKey
+	nodeStrategies := make([]testutils.CountToStrategy, 0, 10)
+	for i := 0; i < numGroups; i++ {
+		nodeStrategies = append(nodeStrategies, testutils.CountToStrategy{
+			Count:    numNodes / numGroups,
+			Strategy: testutils.NewLabelNodePrepareStrategy(nodeAffinityKey, fmt.Sprintf("%v", i)),
+		})
+	}
+	return nodeStrategies
+}
+
+func (nodeAffinity *nodeAffinity) mutatePod(numPods int, pod *v1.Pod) []*v1.Pod {
+	numGroups := nodeAffinity.numGroups
+	nodeAffinityKey := nodeAffinity.nodeAffinityKey
+	podList := make([]*v1.Pod, 0, numGroups)
+	for i := 0; i < numGroups; i++ {
+		pod = &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "sched-perf-node-affinity-pod-",
+			},
+			Spec: testutils.MakePodSpec(),
+		}
+		pod.Spec.Affinity = &v1.Affinity{
+			NodeAffinity: &v1.NodeAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+					NodeSelectorTerms: []v1.NodeSelectorTerm{
+						{
+							MatchExpressions: []v1.NodeSelectorRequirement{
+								{
+									Key:      nodeAffinityKey,
+									Operator: nodeAffinity.Operator,
+									Values:   []string{fmt.Sprintf("%v", i)},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+		podList = append(podList, pod)
+	}
+	return podList
+}
+
+// Interface that every predicate or priority structure should implement.
```

Comment:
wait, why do these functions `mutateNode` and `mutatePod` not match the signatures in the structs?